app/routes/task_routes.py

```diff
@@ -0,0 +1,116 @@
+from app import db
+from app.models.task import Task
+from .route_helpers import *
+from flask import Blueprint, request, make_response, jsonify
+import datetime
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@bp.route("", methods=["GET"])
+def get_tasks():
+    sort_query = request.args.get("sort")
+    search_title_query = request.args.get("search_title")
+    if sort_query:
+        if sort_query == "asc":
+            tasks = Task.query.order_by(Task.title).all()
+        elif sort_query == "desc":
+            tasks = Task.query.order_by(Task.title.desc()).all()
+        else:
+            return make_response(f"{sort_query} is not a valid sort parameter. Please use sort=asc or sort=desc.", 400)
+    elif search_title_query:
+        tasks = Task.query.filter_by(title=search_title_query).all()
+    else:
+        tasks = Task.query.all()
+        
+    return jsonify([task.to_json() for task in tasks])
+
+@bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    for attribute in {"title", "description", "completed_at"}:
+        if attribute not in request_body:
+            return make_response({"details": "Invalid data"}, 400)   
```

Comment:
Nice way to check for a set of required fields. This could be pulled out into a validation method as you did for the the completed_at date string, and a similar approach to my comment about error handling in the `route_helpers` could turn this into a fire and forget method you could use here and in your PUT handler.